### PR TITLE
SRVCOM-1580: docs cleanup

### PR DESCRIPTION
**Pull request by abrennan89**
Applies for OCP 4.6+

- Cleans up unnecessary fluff.
- Updates to simplify structure.

Jira: https://issues.redhat.com/browse/SRVCOM-1580

Direct preview links:
- Update: https://deploy-preview-40094--osdocs.netlify.app/openshift-enterprise/latest/serverless/serverless-update.html
- Serverless apps: https://deploy-preview-40094--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-applications.html
- Installing kn: https://deploy-preview-40094--osdocs.netlify.app/openshift-enterprise/latest/serverless/cli_tools/installing-kn.html
- About Serverless: https://deploy-preview-40094--osdocs.netlify.app/openshift-enterprise/latest/serverless/discover/about-serverless.html

### CODE REVIEW

File: serverless/serverless-update.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+toc::[]
+
+If you have installed a previous version of {ServerlessProductName}, you can follow the instructions in this guide to update to the latest version.
+
+[IMPORTANT]
+====
+Before updating to the latest {ServerlessProductName} release, you must remove the community Knative Eventing Operator if you have previously installed it.
+
+Having the Knative Eventing Operator installed prevents you from being able to install the latest version of Knative Eventing by using the {ServerlessOperatorName}.
```
Comment (michaelryanpeter) [inline serverless/serverless-update.adoc:15]: > Having the Knative Eventing Operator installed prevents you from being able to install the latest version of Knative Eventing by using the {ServerlessOperatorName}.

This sentence is throwing me a little. What do you think about rephrasing it a bit?

"If an earlier version of the Knative Eventing Operator is installed, you cannot install the latest version of the Operator using {ServerlessOperatorName}."

**Review by michaelryanpeter — COMMENTED** [review]
Looks good! Just a few small comments.

File: serverless/discover/about-serverless.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+:context: about-serverless
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Serverless is a cloud computing model where application developers do not need to provision servers or manage scaling for their applications. These routine tasks are abstracted away by the platform, allowing developers to push code to production more quickly than in traditional models.
+
+{ServerlessProductName} simplifies the process of delivering code from development into production by reducing the need for infrastructure set up or back-end development by developers.
+
+Developers on {ServerlessProductName} can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads, by creating Kubernetes custom resources.
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply serverless/discover/about-serverless.adoc:13]: This text already existed but I had just added the last "by creating Kubernetes custom resources." bit when I moved it. On reading this back though I've decided to remove that and go with the original sentence.

> Developers on {ServerlessProductName} can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.

File: serverless/discover/about-serverless.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+{ServerlessProductName} is based on the open source link:https://knative.dev/docs/[Knative project], which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply serverless/discover/about-serverless.adoc:15]: perfect, thank you!

File: serverless/serverless-update.adoc (hunk 1 above)
Comment (abrennan89) [reply serverless/serverless-update.adoc:15]: Reworded this, thanks

**Pull request merged**